src/components/molecules/ScheduleVenueDescription/ScheduleVenueDescription.tsx

```diff
@@ -34,15 +34,15 @@ export const ScheduleVenueDescription: FC<ScheduleVenueDescriptionProps> = ({
   );
 
   const { subtitle, description } =
-    sovereignVenue?.config?.landingPageConfig ?? {};
+    currentVenue?.config?.landingPageConfig ?? {};
 
   return (
     <div className={containerClasses}>
       <div className="ScheduleVenueDescription__main">
         <div className="ScheduleVenueDescription__pic" />
         <div className="ScheduleVenueDescription__title">
           <h2 className="ScheduleVenueDescription__name">
-            {sovereignVenue?.name ?? "Schedule"}
+            {currentVenue?.name ?? "Schedule"}
```

Comment:
Maybe extract a constant
```
const DEFAULT_SCHEDULE_NAME = "Schedule";
```

Reply:
fix in fc6549ba7fe1cff05c89fdbbd261def4decd3b54